### ISSUE TITLE
google.cloud.error_reporting AttributeError: 'Client' object has no attribute '_project'

### ISSUE THREAD

**Issue by DizzeePascall** [GoogleCloudPlatform/google-cloud-python#3233]
When running the examples from the documentation, I get the following error: `AttributeError: 'Client' object has no attribute '_project'`

Disabling GRPC via the GOOGLE_CLOUD_DISABLE_GRPC environment variable fixes this.

I'm running on an Ubuntu 14.04 compute instance with python 3.5.3 and version 023.2 of the google-cloud-error-reporting project

**Comment by DizzeePascall** [comment]
The problem seems to exist because PRs #3158 and #3159 have interefered with each other

**Comment by dhermes** [comment]
@DizzeePascall We just pushed `0.23.2` to fix this issue. Can you try updating?

**Comment by DizzeePascall** [comment]
@dhermes I'm running `0.23.2`. As I mentioned, two PRs were filed to address a similar problem in `0.23.1`, but ironically they've cancelled each other out. This shouldn't be closed yet

**Comment by dhermes** [comment]
Thanks for the heads up. Can you post the snippet you used that failed so we can confirm as well? (You like to one PR and one issue, maybe you meant to link to two PRs?)

**Comment by DizzeePascall** [comment]
@dhermes Good point, I've updated my comment above

**Comment by dhermes** [comment]
Confirming the issue:

```
$ git grep -n '\._project' -- error_reporting/
error_reporting/google/cloud/error_reporting/_gax.py:44:    return _ErrorReportingGaxApi(gax_client, client._project)
error_reporting/google/cloud/error_reporting/_gax.py:60:        self._project = project
error_reporting/google/cloud/error_reporting/_gax.py:73:        project_name = self._gax_api.project_path(self._project)
error_reporting/tests/unit/test__gax.py:55:        self.assertIs(report_error_client._project, client._project)
error_reporting/tests/unit/test__gax.py:71:        self.assertEqual(gax_client_wrapper._project, self.PROJECT)
$ # WHERE/WHEN
$ git log -1 --pretty=%H
e63ddc384e4a54a6a085595d779a399ed449c26c
```

**Comment by asilversempirical** [comment]
Any ETA on a fix for this? My team's error reporting is broken on this. Should we just temporarily pin to .23.0?

**Comment by dhermes** [comment]
@asilversempirical Hopefully `0.24.0` (not `0.23.2`) will come out this afternoon via #3245

**Comment by asilversempirical** [comment]
Excellent, thank you for being on top of this :)

**Comment by lukesneeringer** [comment]
We missed yesterday afternoon, but it is coming out this morning.

**Comment by lukesneeringer** [comment]
This issue should be fixed in 0.24.0.
